index.js

```diff
@@ -0,0 +1,18 @@
+const innerImportVisitor = {
+  StringLiteral(path) {
+    const packageName = path.node.value;
+    if (packageName === '@glimmer/component') {
+      path.node.value = '@ember/component';
+    }
+  }
+};
+
+module.exports = function() {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        path.traverse(innerImportVisitor);
```

Comment:
This is going to need to be a bit more sophisticated. Consider these cases:

```js
import Component, { tracked } from '@glimmer/component';
```

Will become:

```js
import Component, { tracked } from '@ember/component';
```

This is not what we are going to want as we don't want the Component import to be changed.

Instead we want to emit.

```js
import Component from '@glimmer/component';
import { tracked } from '@ember/component';
```

We actually need to look which `ImportSpecifier`s are being imported and only emit the new declaration if `tracked` is imported.

Reply:
This should be working now. Let me know if there are any other edge cases I should be addressing.